codebundles/aws-c7n-ec2-health/sli.robot

```diff
@@ -0,0 +1,85 @@
+*** Settings ***
+Metadata          Author   saurabh3460
+Metadata          Supports    AWS    EC2    CloudCustodian
+Metadata          Display Name    AWS EC2 Health
+Documentation     Count the number of EC2 instances with degraded health status in a specified AWS region
+Force Tags    EC2    Compute    AWS
+
+Library    RW.Core
+Library    RW.CLI
+Library    CloudCustodian.Core
+
+Suite Setup    Suite Initialization
+
+
+*** Tasks ***
+Check for old AWS EC2 instances in AWS Region `${AWS_REGION}` in AWS account `${AWS_ACCOUNT_ID}` 
+    [Documentation]  Check for old EC2 instances in AWS Region. 
+    [Tags]    ec2    instance    aws    compute
+    ${result}=    CloudCustodian.Core.Generate Policy   
+    ...    ${CURDIR}/old-ec2-instances.j2    
+    ...    days=${AWS_EC2_AGE}    
+    ...    state=${AWS_EC2_STATE}    
+    ...    tags=${AWS_EC2_TAGS}
+    ${c7n_output}=    RW.CLI.Run Cli
+    ...    cmd=custodian run -r ${AWS_REGION} --output-dir ${OUTPUT_DIR}/aws-c7n-ec2-health ${CURDIR}/old-ec2-instances.yaml --cache-period 0
+    ...    secret__aws_access_key_id=${AWS_ACCESS_KEY_ID}
+    ...    secret__aws_secret_access_key=${AWS_SECRET_ACCESS_KEY}
+    ${count}=     RW.CLI.Run Cli
```

Comment:
This likely is more complicated that it needs to be. Why evaluate a 1 or a 0 when this is a single check - the total count is a fine metric to push, and then SLI alerts can be configured as needed. This only makes sense as a 1 or a 0 if you want to perform an aggregate score across multiple tasks. Do you plan on adding more tasks?

Reply:
next code bundle like `AutoScaling Group - Verify ASGs have valid configurations` comes under EC2 instance so I was thinking to include in this EC2 health codebundle